src/runtime/route-rules.ts

```diff
@@ -46,10 +46,22 @@ export function createRouteRulesHandler() {
         const query = getQuery(event.path);
         target = withQuery(target, query);
       }
-      return proxyRequest(event, target, {
-        fetch: $fetch.raw as any,
-        ...routeRules.proxy,
-      });
+
+      const transparentOptions: any = {}
+      if (routeRules.proxy.transparent) {
+        transparentOptions.fetchOptions = {
+          ignoreResponseError: true
```

Comment:
We can pass it by default from h3 assuming nitro and others might pass `$fetch.raw` as implementation as a fix